src/components/cart/price-card.jsx

```diff
@@ -1,19 +1,32 @@
 import React from 'react';
+import { useCart } from '../../contexts/cart-context';
 import './price-card.css'
 
 
 const PriceCard = () => {
+  const{cart}=useCart();
+  const totalBill = cart.reduce((acc, curr) => {
+    return acc + curr.quantity * curr.discountedPrice;
+  }, 0);
   return (
     <div class="card-wrapper checkout-wrapper flex-col">
         <h2 class="wishlist-header">Price Details</h2>
         <hr class="checkout-gap" />
         <div class="justify-content-sb dis-flex">
-          <p class="check-out-text">Price(1 item)</p>
-          <p class="check-out-text">Rs 5000</p>
+          <p class="check-out-text">Price({cart.length} item)</p>
+          <p class="check-out-text">
+         Rs{cart.reduce((acc, curr) => {
+         return acc + curr.quantity * curr.price;
+         }, 0)}
+          </p>
         </div>
         <div class="justify-content-sb dis-flex">
           <p class="check-out-text">Discount</p>
-          <p class="check-out-text">-Rs 4000</p>
+          <p class="check-out-text">
+          Rs{cart.reduce((acc, curr) => {
+          return acc + curr.quantity *( curr.price-curr.discountedPrice);
```

Comment:
Just a suggestion to use a single reduce method to get all the values you need

Reply:
sounds good...I will look into it